riscv-c-api.md

```diff
@@ -318,6 +318,12 @@ __attribute__((target("arch=+v"))) int foo(void) { return 0; }
 __attribute__((target("arch=+zbb"))) int foo(void) { return 1; }
 ```
 
+### `__attribute__((riscv_vector_cc))`
+
+Functions declared with this attribute will use to the standard vector calling
+convention variant as defined in the RISC-V psABI, even if the function has
+vector arguments or returing value.
```

Comment:
typo: "returing"
Proposal: "...has vector arguments or a return value".

Reply:
Oh, damm, it's the only word not reviewed by my grammar checker :P